src/Migrations/Version20200907102535.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20200907102535 extends AbstractMigration
```

Comment:
It would be nice to use doctrine migrations 3.0 after this PR https://github.com/Sylius/PayPalPlugin/pull/163, but probably in the separate PR

Reply:
that's correct 👍